### DISCUSSION TITLE
Can I set resource requests and limits in RecurringJobs?

### DISCUSSION THREAD

**Discussion by mrclrchtr** [longhorn/longhorn#9200] [Q&A] [answered]
As in title, is it possible to set resource requests and limits for the jobs?

**Comment by ChanYiLin** [comment, marked as answer]
Hi, currently there is no place for users to set the resource requests and limits for the jobs.
Moreover, I have checked the code, even if you update the CronJob Spec manually with kubectl, Longhorn will periodically update the CronJob Spec back.
I think this is worth implementing, I will create a ticket from this discussion so we can tackle this feature in the near future.

**Reply by mrclrchtr** [reply]
Thank you very much for the fast response! 
Yes, I think that would be a good thing.

**Comment by ChanYiLin** [comment]
Here is the ticket: https://github.com/longhorn/longhorn/issues/9168

**Reply by mrclrchtr** [reply]
I think you mean #9201

**Reply by ChanYiLin** [reply]
Yes you are right!
Thanks for the correction, it is https://github.com/longhorn/longhorn/issues/9201 indeed.
